### PR TITLE
Update Curation Concerns

### PR DESCRIPTION
**Pull request by tpendragon**
This updates to the latest curation concerns, which adds new ordering
from ActiveFedora::Aggregations and the new interface for
Hydra::Derivatives.

### CODE REVIEW

File: spec/factories/generic_files.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FactoryGirl.define do
```
Comment (escowles) [inline spec/factories/generic_files.rb]: Do you want to rename the file to file_sets.rb to keep it in sync with the factory name?

File: spec/support/features/session_helpers.rb [hunk 2]
```diff
@@ -6,6 +6,7 @@ def login_as(user)
       user.reload # because the user isn't re-queried via Warden
       super(user, scope: :user, run_callbacks: false)
     end
+
```
Comment (escowles) [inline spec/support/features/session_helpers.rb]: Super minor, but you could back out this whitespace change if you're making other changes.

Comment (tpendragon) [inline spec/support/features/session_helpers.rb]: I'd love to, but rubocop complains.

File: spec/factories/generic_files.rb (hunk 1 above)
Comment (tpendragon) [inline spec/factories/generic_files.rb]: Done

File: spec/support/features/session_helpers.rb (hunk 2 above)
Comment (escowles) [inline spec/support/features/session_helpers.rb]: OK, that's pretty weird that it passed before then.

**Pull request merged**